ProcessLib/GroundwaterFlowProcess.h

```diff
@@ -260,16 +306,33 @@ class GroundwaterFlowProcess : public Process
 
         // Apply known values from the Dirichlet boundary conditions.
         MathLib::applyKnownSolution(*_A, *_rhs, _dirichlet_bc.global_ids, _dirichlet_bc.values);
-
         typename GlobalSetup::LinearSolver linearSolver(*_A);
         linearSolver.solve(*_rhs, *_x);
-
+#endif
         return true;
     }
 
     void post(std::string const& file_name)
     {
         DBUG("Postprocessing GroundwaterFlowProcess.");
+
+#ifdef USE_PETSC // Note: this is only a test
+        int rank;
```

Comment:
As mentioned before, this output is only for test purpose, and there will be another PR for a parallel VTK output.